### PR TITLE
[no ticket] Make create namespace API async a few other small fixes

### PR DESCRIPTION
**Pull request by Qi77Qi**
Tested via local swagger.

### CODE REVIEW

File: openapi/src/parts/controlled_azure_relay_namespace.yaml [hunk 1]
```diff
@@ -20,6 +20,24 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/relayNamespace/create-result/{jobId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/JobId'
+    get:
+      summary: Retrieve information about an Azure Vm create job.
```
Comment (aherbst-broad) [inline openapi/src/parts/controlled_azure_relay_namespace.yaml:28]: ⛏️ There are a few allusions in the swagger changes to creating a VM, but we're creating a relay namespace right?

File: service/src/main/java/bio/terra/workspace/service/resource/ResourceValidationUtils.java [hunk 2]
```diff
@@ -224,7 +224,7 @@ public static void validateAzureNamespace(String name) {
     if (!AZURE_RELAY_NAMESPACE_PATTERN.matcher(name).matches()
         || name.length() > 50
         || name.length() < 6) {
-      logger.warn("Invalid Azure Namespace", name);
+      logger.warn("Invalid Azure Namespace {}", name);
```
Comment (aherbst-broad) [inline service/src/main/java/bio/terra/workspace/service/resource/ResourceValidationUtils.java:227]: 👍 for the better diagnostic logging

File: openapi/src/parts/controlled_azure_relay_namespace.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ paths:
     parameters:
       - $ref: '#/components/parameters/WorkspaceId'
     post:
-      summary: Create a new controlled Azure Relay Namespace
+      summary: Asynchronoulsy create a new controlled Azure Relay Namespace
```
Comment (aherbst-broad) [inline openapi/src/parts/controlled_azure_relay_namespace.yaml:6]: Very small nitpick: 
```suggestion
      summary: Asynchronously create a new controlled Azure Relay Namespace
```

**Review by aherbst-broad — APPROVED** [review]
Lgtm, I like the idea of making this an async op.

File: openapi/src/parts/controlled_azure_relay_namespace.yaml [hunk 4]
```diff
@@ -102,26 +120,38 @@ components:
...
+    CreatedControlledAzureRelayNamespace:
+      description: Api result class for creating A new Azure VM
       type: object
       properties:
         resourceId:
           description: UUID of a newly-created resource.
           type: string
           format: uuid
-        azureRelayNameSpace:
+        azureVm:
```
Comment (ddietterich) [inline openapi/src/parts/controlled_azure_relay_namespace.yaml:134]: Did you mean to do this?

**Comment by ddietterich** [comment]
> {
  "message": "bio.terra.stairway.exception.JsonConversionException: Failed to deserialize value '[\"bio.terra.workspace.service.resource.controlled.cloud.azure.relayNamespace.ControlledAzureRelayNamespaceResource\",{\"workspaceId\":\"ba5d034e-ae3f-4c4a-aa8d-7c0d004352b5\",\"resourceId\":\"211f0f0d-ce2f-4418-b154-c51781826bc1\",\"name\":\"c-relay-ns\",\"description\":\"string\",\"cloningInstructions\":\"COPY_NOTHING\",\"assignedUser\":null,\"privateResourceState\":\"NOT_APPLICABLE\",\"accessScope\":\"ACCESS_SCOPE_SHARED\",\"managedBy\":\"MANAGED_BY_USER\",\"applicationId\":null,\"namespaceName\":\"qi-wsm-test\",\"region\":\"westcentralus\",\"uniquenessCheckAttributes\":[\"bio.terra.workspace.db.model.UniquenessCheckAttributes\",{\"uniquenessScope\":\"WORKSPACE\",\"parameters\":[\"java.util.ArrayList\",[[\"org.apache.commons.lang3.tuple.ImmutablePair\",{\"Name\":\"c-relay-ns\"}]]]}],\"resourceFamily\":\"AZURE_RELAY_NAMESPACE\",\"resourceType\":\"CONTROLLED_AZURE_RELAY_NAMESPACE\",\"stewardshipType\":\"CONTROLLED\",\"category\":\"USER_SHARED\"}]' from JSON to type bio.terra.workspace.generated.model.ApiCreatedControlledAzureRelayNamespace",
  "statusCode": 500,
  "causes": []
}

Looking at the JSON, I see an array of `org.apache.commons.lang3.tuple.ImmutablePair. I do not think Jackson can process the Apache immutable type. Not sure where that is coming from, but suggest you use a different datatype for the pair.

**Review by rtitle — APPROVED** [review]
LGTM

**Pull request merged**